gradio/components/chatbot.py

```diff
@@ -263,6 +263,7 @@ def __init__(
         allow_file_downloads=True,
         group_consecutive_messages: bool = True,
         allow_tags: list[str] | bool = True,
+        collapse_thinking: list[tuple[str, str]] | None = None,
```

Comment:
`collapse_thinking` doesn't seem like the right variable name - I would have expected this to be a boolean given the variable name, meaning whether or not the thinking traces should be collapsed. A better name might be "reasoning_tag"? and `<thinking>` seems to be pretty much the standard way, are there any others you've seen?